### PR TITLE
Fixed bad image URL on meta tags

### PR DESCRIPTION
**Pull request by joaopgrassi**

### CODE REVIEW

File: src/NuGetTrends.Spa/src/index.html [hunk 1]
```diff
@@ -23,11 +23,11 @@
   <meta property="twitter:title" content="NuGet Trends" />
   <meta property="twitter:site" content="https://nugettrends.com" />
   <meta property="twitter:description" content="Check out NuGet packages adoption and what's trending on NuGet." />
-  <meta property="twitter:image:src" content="assets/images/banner_social.jpg" />
+  <meta property="twitter:image:src" content="https://nugettrends.com/assets/images/banner_social.png" />
   <meta property="twitter:image:width" content="1200" />
   <meta property="twitter:image:height" content="630" />
   
-  <link rel="image_src" type="image/jpeg" href="assets/images/banner_social.jpg" />
+  <link rel="image_src" type="image/jpeg" href="https://nugettrends.com/assets/images/banner_social.png" />
```
Comment (bruno-garcia) [inline src/NuGetTrends.Spa/src/index.html:30]: It really has to be FQDN? This will be messy when we deploy staging before prod

Comment (bruno-garcia) [reply src/NuGetTrends.Spa/src/index.html:30]: Also the `type` now doesn't match the extensions.

Comment (joaopgrassi) [reply src/NuGetTrends.Spa/src/index.html:30]: I only found for the Open Graph protocol, but looking around what others do, apparently yes

> og:image - An image URL which should represent your object within the graph.

And they have a description of "Types" http://ogp.me/#url

Where `URL` is
```
A sequence of Unicode characters that identify an Internet resource. 
All valid URLs that utilize the http:// or https:// protocols
```
These are just for social media stuff.. and I guess we will not change these once in production.

Comment (bruno-garcia) [reply src/NuGetTrends.Spa/src/index.html:30]: fair enough.

**Pull request merged**